src/test/java/uk/ac/ebi/eva/commons/mongodb/repository/VariantRepositoryTest.java

```diff
@@ -275,6 +276,7 @@ public void testRegionIsFoundWithPolyphenGreaterThan() {
         testFiltersHelperRegion(regions, filters, exclude, 4);
     }
 
+    @Ignore
```

Comment:
These are the ignores because those searches have to be reimplemented for the new database structure, which is Tom task. We talked about it in the standup yesterday.